srv/components/controller/files/scripts/controller-cli.sh

```diff
@@ -523,8 +523,8 @@ main()
     if [[ "$update_fw" = true || "$restart_ctrl_opt" = true
         || "$shutdown_ctrl_opt" = true || "$ntp_opt" = true
         || "$show_license" = true ]]; then
-        echo "main(): Decrypting the password received from api" >> $logfile
-        pass=`salt-call lyveutils.decrypt storage ${pass} --output=newline_values_only`
+        #echo "main(): Decrypting the password received from api" >> $logfile
+        #pass=`salt-call lyveutils.decrypt storage ${pass} --output=newline_values_only`
```

Comment:
Don't change it here.
If possible, we should find a way to make direct calls to the targeted method instead of commenting this here.